packages/components/src/models.ts

```diff
@@ -212,6 +213,10 @@ export function getStateQueryGridModel(
                 modelProps.showViewSelector = props.showViewSelector;
             }
 
+            if (props.showExport !== undefined) {
+                modelProps.showExport = props.showExport;
```

Comment:
Depending on when my open PR gets merged you may need to remove this since I get rid of this block of ifs. If this PR gets merged first I'll take care of it in my branch.

Reply:
Ok we can coordinate